src/components/deploymentConfig/DeploymentTemplateView/DeploymentConfigFormCTA.tsx

```diff
@@ -53,6 +58,17 @@ export default function DeploymentConfigFormCTA({
             </Tippy>
         )
     }
+    
+    const checkForLockedChangesForApproval = async () => {
+        if (isApprovalPending && !approveDisabled && !isSuperAdmin) {
+            const res = await checkForProtectedLockedChanges()
```

Comment:
We should add a try catch here to handle error cases

Reply:
done